doc/doc/basics/extractors.md

```diff
@@ -17,10 +17,15 @@ class contains different extractor *styles*:
 
 - Row-wise extractors:
   - [`json_extractor`](#json_extractor): highly flexible and compatible with
-  previous systems, but with limited performance 
+  previous systems, but with limited performance
```

Comment:
Could you please keep the trailing whitespace cleanup in a separate PR?  It's really hard to see what's really changed.  They should've not introduced in the first place, but I think it's even worse if we clean them up in random commits.